app/src/main/java/uk/gov/android/securestore/SharedPrefsStore.kt

```diff
@@ -23,31 +23,39 @@ class SharedPrefsStore(
 ) : SecureStore {
     private val sharedPrefs = context.getSharedPreferences(configuration.id, Context.MODE_PRIVATE)
 
-    override suspend fun upsert(key: String, value: String): String {
+    override suspend fun upsert(key: String, value: String, context: FragmentActivity): String {
+        authenticator.init(context)
```

Comment:
I think it's a good proactice to move the `init` into the `try`.

You `init` it in the `try`, and you close it in `finally`